src/pages/[language]/blog/page/[page].vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <main class="page-blog">
+    <page-header
+      heading="byline"
+      :byline="data.page.title"
+      :headline="data.page.subtitle"
+      :image="data.page.headerIllustration"
+    />
+
+    <section class="page-blog-container grid">
```

Comment:
On page navigation, can we directly scroll the viewport to this element? then the users don't have to scroll down each time they click on a page in the nav. We could solve this by giving this element an `id` and adding that `id` in the hash of the urls in the nav.

What do you think of this?

Reply:
Ok, I tried to keep it generic in the `pagination-nav` component so it can be reused. But it also required changes to the `app-link` otherwise we lose the hash after adding the trailing slash (this also happened on the version that used query params).

Reply:
I had to make some changes to the `app-link` for it to work while keeping the `pagination-nav` as generic as possible, let me know what you think.